README.pl.md

```diff
@@ -0,0 +1,71 @@
+# sway
+
+[**English**](https://github.com/swaywm/sway/blob/master/README.md#sway--) - [日本語](https://github.com/swaywm/sway/blob/master/README.ja.md#sway--) - [Deutsch](https://github.com/swaywm/sway/blob/master/README.de.md#sway--) - [Ελληνικά](https://github.com/swaywm/sway/blob/master/README.el.md#sway--) - [Français](https://github.com/swaywm/sway/blob/master/README.fr.md#sway--) - [Українська](https://github.com/swaywm/sway/blob/master/README.uk.md#sway--) - [Italiano](https://github.com/swaywm/sway/blob/master/README.it.md#sway--) - [Português](https://github.com/swaywm/sway/blob/master/README.pt.md#sway--) -
+[Русский](https://github.com/swaywm/sway/blob/master/README.ru.md#sway--) - [Български](https://github.com/swaywm/sway/blob/master/README.bg.md#sway--) - [Español](https://github.com/swaywm/sway/blob/master/README.es.md#sway--) -
+[Polski](https://github.com/swaywm/sway/blob/master/README.pl.md#sway--)
+
+sway jest kompozytorem [Wayland](http://wayland.freedesktop.org/) kompatybilnym z i3.
+Przeczytaj [FAQ](https://github.com/swaywm/sway/wiki). Dołącz do [kanału IRC](http://webchat.freenode.net/?channels=sway&uio=d4)
+(#sway na irc.freenode.net).
+
+Jeśli chcesz wesprzeć rozwój sway, rozważ wsparcie SirCmpwn na jego [stronie Patreon](https://patreon.com/sircmpwn).
+
+## Podpisy cyfrowe wydań
+
+Wydania są podpisywane przy pomocy klucza [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
+i publikowane [na GitHubie](https://github.com/swaywm/sway/releases).
+
+## Instalacja
+
+### Z pakietów
+
+Sway jest dostępny w wielu dystybucjach Linuksa. Spróbuj zainstalować pakiet "sway" w swoim
```

Comment:
Sway is available on non-linux distros as well (e.g. FreeBSD), so and original document doesn't specify Linux so I'd omit that here